code/game/objects/items/twohanded.dm

```diff
@@ -902,16 +902,24 @@
 	if (locate(/obj/structure/table) in target.contents)
 		return
 	var/i = 0
+	var/turf/target_turf = get_step(target, user.dir)
+	var/obj/machinery/disposal/bin/target_bin = locate(/obj/machinery/disposal/bin) in target_turf.contents
 	for(var/obj/item/garbage in target.contents)
 		if(!garbage.anchored)
-			garbage.Move(get_step(target, user.dir), user.dir)
-		i++
-		if(i >= 20)
+			if (target_bin)
+				garbage.forceMove(target_bin)
```

Comment:
You could make this give a little janitor xp for each piece of garbage

Reply:
its a slippery slope because it would be ludicrously easy to farm xp in a very naughty way i think

Reply:
but i do like the idea hmm